jbpm-workitems/pom.xml

```diff
@@ -141,7 +141,6 @@
     <dependency>
       <groupId>org.apache.cxf</groupId>
       <artifactId>cxf-rt-transports-http</artifactId>
-      <scope>runtime</scope>
     </dependency>
     <dependency>
       <groupId>wsdl4j</groupId>
```

Comment:
Please include also wsdl4j into compile scope, otherwise it is failing with:

```Compilation failure
required: List<File>
found:    Object
/home/jenkins/workspace/jbpm-pullrequests-6.5.x/jbpm-workitems/src/main/java/org/jbpm/process/workitem/webservice/WebServiceWorkItemHandler.java:[252,14] error: cannot access ExtensibilityElement
```